src/databricks/labs/ucx/queries/assessment/estimates/02_2_uc_data_modeling.sql

```diff
@@ -1,3 +1,20 @@
--- viz type=table, name=Tables to migrate, columns=catalog,database,name,object_type,table_format,location,view_text,upgraded_to,storage_properties
+-- viz type=table, name=Tables to migrate, search_by=table_name,upgraded_status, columns=table_name,object_type,table_format,location,upgraded_status,upgraded_to_table_name,view_text,storage_properties,is_partitioned
 -- widget title=Tables to migrate, row=2, col=2, size_x=3, size_y=8
-select * from $inventory.tables;
\ No newline at end of file
+SELECT
+  concat_ws('.', tables.`catalog`, tables.`database`, tables.name) AS table_name,
+  tables.object_type,
+  tables.table_format,
+  tables.location,
+  CASE
+      WHEN migration_status.dst_table IS NOT NULL THEN 'Migrated'
+      ELSE 'Stayed in place'
+  END AS upgraded_status,
+  concat_ws('.', migration_status.dst_catalog, migration_status.dst_schema, migration_status.dst_table) AS upgraded_to_table_name,
+  tables.view_text,
+  tables.storage_properties,
+  tables.is_partitioned
+FROM
+    $inventory.tables AS tables
+  LEFT JOIN
+    $inventory.migration_status AS migration_status
```

Comment:
please attach a screenshot after running the `migrate-tables` workflow. Add (or use) an existing integration test case, that migrates external table and a view.

https://github.com/databrickslabs/ucx/blob/00d2e6e80dfd19c3b83e6557a036093d0b42bb3b/src/databricks/labs/ucx/hive_metastore/workflows.py#L7-L37

here's the test:
https://github.com/databrickslabs/ucx/blob/00d2e6e80dfd19c3b83e6557a036093d0b42bb3b/tests/integration/test_installation.py#L349-L392

perhaps you'd need to do something along the lines of:

```
@job_task(
        job_cluster="table_migration",
        depends_on=[migrate_external_tables_sync, migrate_dbfs_root_delta_tables, migrate_views],
    )
    def refresh_migration_status(self, ctx: RuntimeContext):
        """Refreshes migration status table"""
        ...
```

and you'd have to modify the `MigrateHiveSerdeTablesInPlace` workflow the same way